website/docs/docs/collaborate/project-recommendations.md

```diff
@@ -3,20 +3,20 @@ title: "Project recommendations"
 sidebar_label: "Project recommendations"
 description: "dbt Explorer provides recommendations that you can take to improve the quality of your dbt project."
 ---
-
-:::tip Beta 
-
-The project recommendations beta feature is now available in dbt Explorer! Check it out!
-
-:::
```

Comment:
callouts at the top of pages affect SEO so as a best practice we're moving them to a lower location on the page but making sure they're still before the fold